data/BAL/data.json

```diff
@@ -8,6 +8,9 @@
     },
     "optimism": {
       "address": "0xFE8B128bA8C78aabC59d4c64cEE7fF28e9379921"
+    },
+    "goerli": {
```

Comment:
is there a corresponding address for optimism goerli for this asset? we typically only add l1 addresses that have a l2 asset as well.